webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useMemo, useState } from 'react';
+
+import Icon from 'components/Icon';
+import PageHeaderFoldable, { Option } from 'components/PageHeaderFoldable';
+import ExperimentHeaderFoldableContent
+  from 'pages/ExperimentDetails/Header/ExperimentHeaderFoldableContent';
+import ExperimentHeaderLeft from 'pages/ExperimentDetails/Header/ExperimentHeaderLeft';
+import ExperimentHeaderProgress from 'pages/ExperimentDetails/Header/ExperimentHeaderProgress';
+import { archiveExperiment, openOrCreateTensorboard, unarchiveExperiment } from 'services/api';
+import { getStateColorCssVar } from 'themes';
+import { ExperimentBase } from 'types';
+import { terminalRunStates } from 'utils/types';
+import { openCommand } from 'wait';
+
+interface Props {
+  experiment: ExperimentBase;
+  fetchExperimentDetails: () => void;
+  showForkModal: () => void;
+}
+
+const ExperimentDetailsHeader: React.FC<Props> = (
+  { experiment, fetchExperimentDetails, showForkModal }: Props,
+) => {
+  const [ isRunningArchive, setIsRunningArchive ] = useState<boolean>(false);
+  const [ isRunningTensorboard, setIsRunningTensorboard ] = useState<boolean>(false);
+  const [ isRunningUnarchive, setIsRunningUnarchive ] = useState<boolean>(false);
+
+  useEffect(() => {
+    setIsRunningArchive(false);
+    setIsRunningUnarchive(false);
+  }, [ experiment.archived ]);
+
+  const headerOptions = useMemo<Option[]>(() => {
+    const options: Option[] = [
+      {
+        icon: <Icon name="tensorboard" size="small" />,
```

Comment:
@hkang1 i think the fork icon is missing.

Reply:
ah I'll get this to you